src/frontend/contexts/AppProviders.tsx

```diff
@@ -56,9 +56,7 @@ export const AppProviders = ({
                 <ServerLoading messagePort={messagePort}>
                   <LocalDiscoveryProvider value={localDiscoveryController}>
                     <ApiProvider api={mapeoApi}>
-                      <MetricsProvider
-                        appMetrics={appMetrics}
-                        deviceMetrics={deviceMetrics}>
+                      <MetricsProvider>
```

Comment:
Initializing at `appMetrics` and `deviceMetrics` at the app level still makes sense. You don't have to worry about memoizing it and it is unattached from the react-lifecycle

Reply:
All set I think